packages/validator/src/util/params.ts

```diff
@@ -1,20 +1,27 @@
-import {IChainConfig, chainConfigToJson} from "@chainsafe/lodestar-config";
+import {IChainConfig, chainConfigToJson, defaultChainConfig} from "@chainsafe/lodestar-config";
 
 export class NotEqualParamsError extends Error {}
 
 /**
  * Assert localConfig values match externalSpecJson. externalSpecJson may contain more values than localConfig.
  */
 export function assertEqualParams(localConfig: IChainConfig, externalSpecJson: Record<string, string>): void {
-  const params1Json = chainConfigToJson(localConfig) as Record<string, unknown>;
-  const params2Json = externalSpecJson;
+  const localConfigJson = chainConfigToJson(localConfig) as Record<string, unknown>;
+
+  const remoteConfigJson = {
+    // fill missing properties in remote config with spec default values
+    ...chainConfigToJson(defaultChainConfig),
+    ...externalSpecJson,
+  };
 
   const errors: string[] = [];
 
   // Ensure only that the localConfig values match the remote spec
-  for (const key of Object.keys(params1Json)) {
-    if (params1Json[key] !== params2Json[key])
-      errors.push(`${key} different value: ${params1Json[key]} != ${params2Json[key]}`);
+  for (const key of Object.keys(localConfigJson)) {
+    if (
+      (localConfigJson[key] as string)?.toLocaleLowerCase() !== (remoteConfigJson[key] as string)?.toLocaleLowerCase()
```

Comment:
Maybe better
```ts
const localValue = String(localConfigJson[key]).toLocaleLowerCase()
const remoteValue = String(remoteConfigJson[key]).toLocaleLowerCase()
       if (localValue !== remoteValue) {
         errors.push(`${key} different value: ${localConfigJson[key]} != ${remoteConfigJson[key]}`);
       }
```

Reply:
Sorry forgot to substitute `remoteValue` in the log

Reply:
But since it's not what's being used to compare the error will be very confusing if String() coercion does something stupid

Reply:
updated as suggested